messages/format.go

```diff
@@ -0,0 +1,16 @@
+package messages
+
+import (
+	"fmt"
+)
+
+// FormatAmount formats a cent amount as dollars.
+func FormatAmount(v int64) string {
```

Comment:
This can't be done because it is not in the same package that uses it. I would have to copy it into both the `commands` and `events` packages for it to be unexported.

Reply:
Hmm, I agree, there is no simple way to unexpose this function. 

I thinking about a dedicated monetary type to represent the amount, e, g:
```golang
type Amount int64

// String returns a textual representation of the amount. 
func (a Amount) String() string {
    f := "$%d.%02d"
        if a < 0 {
            a = -a
	    f = "-" + f
	}

	return fmt.Sprintf(f, a/100, a%100)
}
```
You can use this type to assign you amount to and pass to directly to `fmt.Sprintf()` with `%v` formatter in the messages. I would work because your `Amount` implements `fmt.Stringer` interface which all printing functions of `fmt` package favour. 

What do you think guys?

Reply:
You could do that, but that's a bigger change than just "adding descriptions" which is what this PR is about. Another option is to move `FormatAmount()` into an internal package, like `messages/internal/description`.

But I think for the sake of simplicity just leave it as is.